<launch>
	<include file="$(find baxter_chess)/launch/ar_track.launch" />
	<node name="joint_trajectory_action_server" pkg="baxter_interface" type="joint_trajectory_action_server.py" output="screen" />
	<node name="move" pkg="baxter_chess" type="move.py" output="screen" />
	<node name="piece" pkg="baxter_chess" type="piece.py" output="screen" />
	<node name="controller" pkg="baxter_chess" type="control.py" output="screen" />
</launch>